Gemfile

```diff
@@ -176,7 +176,7 @@ group :test, :development do
 
   gem 'rswag-specs'
 
-  gem 'annotate'
+  gem 'annotate', github: 'discourse/annotate_models', branch: 'rails-7'
```

Comment:
The annotate gem has now support for Rails 7 in their `develop` branch but there’s still no new release at the moment :confused: